schemas/context/segmentdefinition.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentdefinition",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Definition",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Segment Definition class to describe a segment from any system",
+  "definitions": {
+    "segmentdefintion": {
+      "properties": {
+        "xdm:segmentIdentity": {
+          "title": "Segment Identity",
+          "$ref": "https://ns.adobe.com/xdm/context/segmentidentity",
+          "description": "Identity of the segment"
+        },
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
+          "description": "Name of the segment"
+        },
+        "xdm:description": {
+          "title": "Description",
+          "type": "string",
+          "description": "Description of the segment"
+        },
+        "xdm:source": {
```

Comment:
I am not sure what the difference between a namespace in segmentIdentity and source is. Can you please help me understand what this source is for?

Reply:
Source is contributing system where as namespace is sub type within the source. Thought segment realization's identity currently has just identity and namespace and no source. That means namespace has to be unique through out all contributing systems. Which makes source redundant. Only benefit it offers is giving information on provider of the segment definition. Having said that, if you think we have/should have other means to relate namespace to source, we can remove source from here.

Reply:
Does this explanation need to be put into the field descriptions?

Reply:
@ankushkd can youupdated the descriptions with an edited version of this explanation?

Reply:
@ktukker  Modified. Please review.

Reply:
@ankushkd It would be good to think about this in context of information present in segment membership. Reason for the ask is that the mapping from namespace to other contextual information may be required to make sense of segment membership info as well.